### ISSUE TITLE
Error when trying to export Working Journal to PDF

### ISSUE THREAD

**Issue by UlrichSi** [Open-Systems-Pharmacology/PK-Sim#1361]
To reproduce, open any project with a working journal, or create a working journal. 
From the File tab, select "Print Preview", then "Export to" and select "PDF file". 
Leave the options on default, click OK. Use the default file name to save.
The following error message is displayed:

`Cannot perform the specified task. The following library is missing: "DevExpress.Pdf.v18.2.Core, Version=18.2.7.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a"
`

![grafik](https://user-images.githubusercontent.com/27352743/69705649-42fb8300-10f6-11ea-8eaf-b4d3569662e7.png)

**Comment by msevestre** [comment]
Yeah. A dll is missing it seems. We should ensure that we update the package once we update the DEvExpress version
